src/parser/vast_parser.js

```diff
@@ -576,4 +580,32 @@ export class VASTParser extends EventEmitter {
       }
     }
   }
+
+  /**
+   * Calculate the estimated bitrate and add it to the bitrates list
+   * @param {Number} byteLength - The length of the response in bytes.
+   * @param {Number} duration - The duration of the request in ms.
+   */
+  updateEstimatedBitrate(byteLength, duration) {
+    if (!byteLength || !duration || byteLength <= 0 || duration <= 0) {
+      return;
+    }
+
+    // We want the bitrate in bit by seconds, byteLength are in bytes and duration in ms, we need to convert them
+    const bitrate = (byteLength * 8) / duration * 1000;
+    this.estimatedBitrates.push(bitrate);
```

Comment:
```suggestion
    const parsedByteLength = parseInt(byteLength, 10);
    const parsedDuration = parseInt(duration, 10);
    if (isNaN(parsedByteLength) || isNaN(parsedDuration) || parsedByteLength <= 0 || parsedDuration <= 0) {
      return;
    }

    // We want the bitrate in bit by seconds, byteLength are in bytes and duration in ms, we need to convert them
    const bitrate = (parsedByteLength * 8) / parsedDuration * 1000;
    this.estimatedBitrates.push(bitrate);
```

WDYT?

Also why not keep as byte rate instead of bit rate?